.github/workflows/gh-pages.yml

```
@@ -194,6 +194,55 @@ jobs:
+  convert-woff2:
+    name: Convert WOFF2
+    runs-on: macos-15
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          sparse-checkout: |
+            fonts
+
+      - name: Install Poetry
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)